src/test/java/SYSC4806/Model/CourseTest.java

```diff
@@ -28,20 +33,37 @@
     private CourseRepository courseRepository;
 
     /**
-     * Setting up dummy information for courses
-     * @throws Exception
+     * Instance of course to add to repository
      */
-    @Before
-    public void setUp() throws Exception {
+    private Course c = new Course(-1, "TEST_COURSE_NAME", "TEST_COURSE_CODE");
```

Comment:
Once model constraints are added, I would assume that this -1 for year would be an illegal argument.